editors/boost-up-my-code-editor.md

```diff
@@ -0,0 +1,96 @@
+# [MO] Boost up my code editor
+
+## Owner: [Guillaume Diallo-Mulliez](https://github.com/guitoof)
+
+## ⛳️ Control Points
+
+My editor allows me to:
+
+### 🐞 **Properly Debug**: so that I can quickly identify the responsibility of an unexpected behaviour
+
+##### What ?
+
+- Setup break points
+- Inspect current context variables
+
+##### How ?
+
+- for **VSCode**: [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
+
+### 🎨 **Automatically Format my Code**: so that I can follow my team syntactic standards and ease up code review and maintanability
+
+##### What ?
+
+- Real time or On save formatting
+- Inspect current context variables
+
+#### How ?
+
+- for **VSCode**: [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+
+### 🔍 **Quickly Spot Syntactic Errors**: so that I don't need to run my linter/type checker commands after each modification
+
+##### What ?
+
+- Linter
+- Type checking (flow, typescript)
+
+#### How ?
+
+- for **VSCode**:
+  - for **ESLint** : [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+  - for **TSLint** : [TSLint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint)
+
+### 📦 **Identify Outdated packages**: so that I can upgrade them rapidly after a new release
+
+##### What ?
+
+- See the latest stable version of each package
+
+#### How ?
+
+- for **VSCode**: [Version Lens](https://marketplace.visualstudio.com/items?itemName=pflannery.vscode-versionlens)
+
+### ☁️ **Visualize my git diff**: so that I can see what I changed since the last time it worked
+
+##### What ?
+
+- See the git diff
+- Revert changes on a file
+
+#### How ?
+
+- for **VSCode**: Build in _Source Control_ tab
+
+### 🛠 **Create & use snippets**: so that I can create components, tests and other basic files in a flash
+
+#### How ?
+
+- for **VSCode**: ???
```

Comment:
Snippets are built-in in VSCode (`cmd+maj+p` then type `snippets`).

However, two remarks:

- writing snippets is a pain. A tool or extension is needed if you want to do it efficiently. I did [a web one](https://pierpo.github.io/vs-snippeteer/) but there might be suitable extensions (which is more convenient)
- there is no built-in way to share snippets. [This extension](https://marketplace.visualstudio.com/items?itemName=rebornix.project-snippets) allows to share snippets in a project!

Reply:
The important part is to share it with your team mates so that you ensure consistency. I added this information in the standard to make it clear thx